packages/components/src/Modal/ConfirmationModal/_docs/ConfirmationModal.stories.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from "react"
+import { Decorator, Meta, StoryObj } from "@storybook/react"
+import isChromatic from "chromatic"
+import { Text } from "~components/Text"
+import { ConfirmationModal, ConfirmationModalProps } from "../index"
+
+const IS_CHROMATIC = isChromatic()
+
+// Add additional height to the stories when running in Chromatic only.
+// Modals have fixed position and would be cropped from snapshot tests.
+// Setting height to 100vh ensures we capture as much content of the
+// modal, as it's height responds to the content within it.
+const HeightDecorator: Decorator<ConfirmationModalProps> = Story => {
+  if (IS_CHROMATIC) {
+    return (
+      <div style={{ minHeight: "100vh" }}>
+        <Story />
+      </div>
+    )
+  }
+
+  return <Story />
+}
+
+const chromaticModalSettings = {
+  parameters: {
+    chromatic: {
+      disable: false,
+      delay: 400, // match MODAL_TRANSITION_TIMEOUT in modals + 50ms
+      pauseAnimationAtEnd: true,
+    },
+  },
+  decorators: [HeightDecorator],
+}
+
+const ConfirmationModalTemplate: Story = {
```

Comment:
@HeartSquared like this...?

Reply:
Yep! This works!

Unsure if it makes the `Playground` a little unclear now, but I suppose if you put `decorators: undefined` (since it's currently still in there) it might be explicit that it's not.

Or maybe it's better to not have the Chromatic settings in the Template, so it's an explicit choice per story as opposed to a default 🤔 
Probably makes this change seem no different to before, but I've definitely had some where I put extra Story properties (eg. `play`) in so it's just a tad more scalable this way.

Reply:
Probably leave it this way for now, I think when we revisit this would be during a rebuild that will hopefully be engineered for stickersheets, and this will all go away 😄